packages/common/src/plugins/types.ts

```diff
@@ -21,14 +21,12 @@ export type Plugin<
    * Use this hook with your own risk. It is still experimental and may change in the future.
    * @internal
    */
-  onRequestParse?: OnRequestParseHook<TServerContext>
+  onRequestParse?: OnRequestParseHook
```

Comment:
Why is this generic no longer necessary?

Reply:
We don't need the server context in OnRequestParse for now. I think we can remove unless we specifically need the server cntext in the parsers.